docs/production/settings.md

```diff
@@ -19,6 +19,10 @@ restart the server with the following command:
 su zulip -c '/home/zulip/deployments/current/scripts/restart-server'
 ```
 
+As this is the main documentation for Zulip settings, we recommend
+[carefully updating your `/etc/zulip/settings.py` every major
+release](../production/upgrade-or-modify.html#updating-configuration-files).
```

Comment:
Maybe this one should provide a bit more context, since it's explaining how settings work overall:
``` quote
`/etc/zulip/settings.py` has useful documentation for dozens of minor settings in its comments. If you've upgraded from an old version of Zulip, your `settings.py` file will not have comments for any settings added since the Zulip release you originally installed; for this reason, we recommend [updating `settings.py`]() after major releases.
```

Reply:
Or maybe just deciding that the existing "miscellaneous" section you have below is good enough, since my suggestion here is basically the exact same text you have there.

Reply:
OK here's a proposal:
* We move "miscellaneous server settings" content to sit here (since its second half just being a new start of "Specific settings" or something).

Reply:
Yeah, I think this works well.